pkgs/os-specific/linux/zenmonitor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk3, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  name = "zenmonitor";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "ocerman";
+    repo = "zenmonitor";
+    rev = "v${version}";
+    sha256 = "10pgw2i8gn8grg6gg43kcipj11lipnd9fyw31jnwrzgwyv4z0w0w";
+  };
+
+  buildInputs = [ gtk3 ];
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+
+  installPhase = ''
```

Comment:
Add TODO comment to use default installPhase in the next release > 1.2:

https://github.com/ocerman/zenmonitor/commit/600680d3ec25de8e6a7816447577ff3c2a5e185d